File: modules/cli/src/test/resources/golden/input/entities/Farbe.scala
package com.breuninger.entdecken.domain.model

import java.util.Locale

case class Farbe(
    id: String,
    farbbezeichnung: Map[Locale, String],
    rgbfilterfarbname: Option[String],
    herstellerfarbe: Option[String]
)

File: modules/cli/src/test/resources/golden/expected/entities/Farbe.kt
package com.breuninger.entdecken.domain.model

import java.util.Locale

data class Farbe(
    val id: String,
    val farbbezeichnung: Map<Locale, String>,
    val rgbfilterfarbname: String?,
    val herstellerfarbe: String?
)
